test/phoenix_component/components_test.exs

```diff
@@ -326,14 +316,12 @@ defmodule Phoenix.LiveView.ComponentsTest do
       </.form>
       """
 
-      html = parse(template)
-
-      assert [
-               {"form", [{"class", "pretty"}, {"data-foo", "bar"}, {"phx-change", "valid"}],
-                [
-                  {"input", [{"id", "base_foo"}, {"name", "base[foo]"}, {"type", "text"}], []}
-                ]}
-             ] = html
+      assert t2h(template) ==
+               ~X"""
+               <form data-foo="bar" class="pretty" phx-change="valid">
+                 <input id="base_foo" name=base[foo] type="text"/>
+               </form>
+               """
```

Comment:
There are other similar tests. I think I prefer this new way, more robust and easier to read from my point of view. Should I change all the tests to be like this one?

Reply:
yes, please update the other tests and this will be good to go!